rpm/kernel-binary.spec.in

```diff
@@ -404,9 +404,9 @@ Requires(post): perl-Bootloader
 Requires(post): dracut
 @PROVIDES_OBSOLETES_EXTRA@
 %obsolete_rebuilds %name-extra
-Supplements:    packageand(product(SLED):%{name}_%_target_cpu)
-Supplements:    packageand(product(sle-we):%{name}_%_target_cpu)
-Supplements:    packageand(product(Leap):%{name}_%_target_cpu)
+Supplements:    (product(SLED) and %{name}_%_target_cpu)
```

Comment:
Since what rpm version is this available?

In particular, dose it work on SLE12 SP5?

Reply:
Requires 4.13+ so no SLE12 support

Reply:
Then it's no good, SLE12 SP5 is still alive.